javascript/chronometer.js

```diff
@@ -1,26 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000)
   }
+
+
   getMinutes() {
-    // ... your code goes here
+    // current time moet naar minuten
+    let minutes = this.currentTime / 60
+    return Math.floor(minutes)
   }
+
   getSeconds() {
-    // ... your code goes here
+    // minutes moeten naar seconden, dus *60
```

Comment:
Good work Sophie! You can make your solution even easier if you use a modulo operator `%` if you're unfamiliar with it, check out [this link](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math))
```
return this.currentTime % 60;